Documentation/ApiOverview/Database/RestrictionBuilder/Index.rst

```diff
@@ -146,6 +146,11 @@ Restrictions
    records need to be handled after the SQL results are fetched, by overlaying the records with
    :php:`BackendUtility::getRecordWSOL()`, :php:`PageRepository->versionOL()` or `PlainDataResolver`.
 
+If a Restriction needs to be enforced as it should always be applied, a restrition could implement the interface `EnforceableQueryRestrictionInterface`. Is this interface implemented for a Restrition, please take care of the following behavior:
```

Comment:
> Is this interface implemented for a Restrition, please _take care of the following behavior_:

I am not sure what is to be done here. To me, what follows is just a clarification what removeAll and removeByType will do. I would phrase it like this:

> If a restriction implements `EnforceableQueryRestrictionInterface`, the following applies.

But this may also be a misunderstanding on my end.